.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,55 @@
+name: "Step 2: Publish Release"
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "The target branch"
+        required: false
+      release_url:
+        description: "The URL of the draft GitHub release"
+        required: false
+      steps_to_skip:
+        description: "Comma separated list of steps to skip"
+        required: false
+        default: "ensure-sha"
```

Comment:
Keeping `ensure-sha` as default here for now as I believe https://github.com/jupyterlab/jupyterlab/issues/13442 is still an issue with Releaser v2 (cc @blink1073 for confirmation).

Reply:
Yes, that is correct